src/dlgTriggerEditor.h

```diff
@@ -178,6 +179,14 @@ class dlgTriggerEditor : public QMainWindow, private Ui::trigger_editor
     void show_vars();
     void setThemeAndOtherSettings(const QString&);
 
+    static const int cmTriggerView;
```

Comment:
🤔 💭 At some point we ought to go the whole way and use an `enum` so that in the public part of the class declarations (before it is used as an argument type) we have:

    enum EditorViewType {
        viewUnknown = 0,
        viewAliases = 0x01,
        viewButtons = 0x02,
        viewKeys = 0x04,
        viewScripts = 0x08,
        viewTimers = 0x10,
        viewTriggers = 0x20,
        viewVariables = 0x40
    }

    Q_DECLARE_FLAGS(EditorViewTypes,EditorViewType)

then below the closing `}` of the class declaration:

    Q_DECLARE_OPERATORS_FOR_FLAGS(dlgTriggerEditor::EditorViewTypes)

then we can _safely_ use bit operations and the like to determine the current view types etc. {and as I have already included other such `enum` in that class the `QFlag`/`QFlags` headers needed are already in place} and we can pass `EditorViewType` as a variable type and an argument type in method calls etc... 😀

Reply:
I'm with you.